css/style.css

```diff
@@ -318,4 +221,4 @@ div.ref_fields {
 .active-site-select {
 	margin-left: 24px;
 	margin-top: 24px;
-}
\ No newline at end of file
+}*/
```

Comment:
I don't think there's an opening comment, so this might break.

Reply:
Removed